common/lib/xmodule/xmodule/capa_base.py

```diff
@@ -200,6 +200,15 @@ class CapaFields(object):
         scope=Scope.settings
     )
 
+    @property
+    def non_editable_metadata_fields(self):
```

Comment:
Maybe, but I'd rather do the simplest thing that could work when it comes to this whole `non_editable_metadata_fields` thing. If you have a suggestion about how to do this cleanly, I'm open to it. We could also ask @cpennington or @doctoryes.